### PR TITLE
Add code for Hardcash

### PR DESCRIPTION
**Pull request by raksha009**
Fixes issue #4503 
Code added for CASH problem of codechef platform

### CODE REVIEW

**Comment by raksha009** [comment]
@arnavb plz review my PR

File: code/online_challenges/src/codechef/CASH/hard_cash.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+class Codechef
```
Comment (arnavb) [inline code/online_challenges/src/codechef/CASH/hard_cash.java]: Code looks fine, but can you just clean up the formatting a bit? Use an online Java formatter.

Comment (raksha009) [reply code/online_challenges/src/codechef/CASH/hard_cash.java]: I cleaned up the formatting. plz review @arnavb

**Comment by arnavb** [comment]
@raksha009 Do you mind if I make some formatting changes locally on my computer? I can fix those issues.

**Comment by raksha009** [comment]
@arnavb  Okk, thanks for helping

**Comment by arnavb** [comment]
1. Please undo the commit where you delete break.c
2. Reformatted java code:

```java
import java.util.Scanner;

class Hardcash {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int t = s.nextInt();

        while (t-- > 0) {
            int n = s.nextInt();
            int k = s.nextInt();
            int[] arr = new int[n];

            //input
            for (int i = 0; i < n; i++) {
                arr[i] = s.nextInt();
            }

            int bag = 0;
            for (int i = 0; i < n; i++) {
                if (arr[i] % k == 0) {
                    continue;
                } else {
                    int step = arr[i] % k;
                    if (k - step <= bag) {
                        bag -= (k - step);
                    } else {
                        bag += step;
                    }
                }
            }
            System.out.println(bag);
        }
    }
}

/*
TEST CASE
2
5 7
1 14 4 41 1
3 9
1 10 19

OUTPUT
5
3
*/

**Pull request closed without merging**